### DISCUSSION TITLE
RSS Feed subscription in Squads

### DISCUSSION THREAD

**Discussion by henriqueleite42** [dailydotdev/daily#1349] [Feature Requests]
Allow squads to subscribe to RSS feeds.

I believe that this would help a lot the transition mentioned on this post:
[https://dly.to/OhrCMUQzaBw](https://dly.to/OhrCMUQzaBw)

**Comment by idoshamun** [comment]
Hi @henriqueleite42,

Thanks for the suggestion, it's a dup of https://github.com/dailydotdev/daily/discussions/778.
Let's keep the discussion there
